spec/contiv/libraries/netmaster.raml

```diff
@@ -1,186 +1,217 @@
 #%RAML 1.0 Library
```

Comment:
Ah okay, 2.0.0 is from like 4-5 years ago, so some things may have changed with YAML in that time.  We'll be using a consistent version from now on so we shouldn't have any more surprisingly large diffs